core/src/main/java/org/mule/runtime/core/internal/event/AbstractEventContext.java

```diff
@@ -150,13 +150,13 @@ public final void success() {
   @Override
   public final void success(CoreEvent event) {
     if (isResponseDone()) {
-      if (LOGGER.isDebugEnabled()) {
+      if (debugLogEnabled) {
         LOGGER.debug(this + " response was already completed, ignoring.");
       }
       return;
     }
 
-    if (LOGGER.isDebugEnabled()) {
+    if (debugLogEnabled) {
       LOGGER.debug(this + " response completed with result.");
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Use the built-in formatting to construct this argument. ([squid:S2629](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS2629))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7701&issues=AWkLOTwXotIXnmsUkh14&open=AWkLOTwXotIXnmsUkh14)

[ISSUE_KEY]: AWkLOTwXotIXnmsUkh14